.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6
+      - image: circleci/postgres:9.5
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: d4s2
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
```

Comment:
One thing I'd like to understand out of these Circle configs is why we chose this key prefix `deps1-` here and when we should modify it. I see that CircleCI uses different conventions for different languages, including [deps9 in their Python examples](https://circleci.com/docs/2.0/language-python/#cache-dependencies) but doesn't seem to explain it.

Reply:
Another way to put it - let's document the reasons we're settling on CircleCI (not just consistency) and the practices we have

Reply:
The documentation is for caching uses various seemingly random prefixes: https://circleci.com/docs/2.0/caching/

Documentation on save cache: https://circleci.com/docs/2.0/configuration-reference/#save_cache

key is `Unique identifier for this cache`
>The cache for a specific key is immutable and cannot be changed once written. NOTE: If the cache for the given key already exists it won’t be modified, and job execution will proceed to the next step.

I think they just had 8 more versions in that branch before they got `deps9`.